### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by umairkarel**
Added some formatting to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ To update the endpoint package mapping take a look at the [Custom Schema](./cust
 
 The essential steps are listed here:
 
-- Edit/add custom schema files as needed to define any fields that don't exist in ECS core
+- Edit/add custom schema files as needed to define any fields that doesn't exist in ECS core
```
Comment (pzl) [inline README.md:32]: the original line was correct, no change needed here

**Review by pzl — CHANGES_REQUESTED** [review]
@umairkarel please sign the Contributor Agreement if you would like us to add your PR

**Review by pzl — APPROVED** [review]
approved. As long as you follow the CLA procedures this can be merged

**Comment by umairkarel** [comment]
> approved. As long as you follow the CLA procedures this can be merged

Definitely, I will follow the procedures.
I had tried to sign the agreement but, there was some problem, the page was not loading.

**Comment by pzl** [comment]
Looks like this has stalled. If you'd like to contribute again, I'm happy to include your changes. Feel free to reopen once the CLA procedure is complete

**Pull request closed without merging**